README.md

```diff
@@ -32,10 +32,34 @@ In addition, EclipseSource also offers [professional support](https://jsonforms.
 
 ### First time setup
 
+If you want to contribute to the project or implement a new renderer components it is recommended to first create a fork of this project so you can later start pull a request to contribute your code.
+
+For a first time setup:
+
 * Install [node.js](https://nodejs.org/) (only Node 14 and npm 6 is currently supported)
+* Fork this repository on Github 
 * Clone this repository
-* Install dependencies: `npm ci`
-* Hook up dependencies between packages: `npm run init`
+
+Now you can build the project and run the examples. First of all you need to install the base dependencies, then run lerna and then build all packages. After that you can execute the example application.
+
+So starting from the root directory:
+
+	# reset and clean everything
+	git reset --hard
+	git clean -dfx
+	
+	# build JSON Forms
+	npm ci
+	npm run init
+	npm run build
+	
+	# e.g. Start React Vanilla example application
+	cd packages/vanilla && npm run dev
+
+Run the example from your web browser: localhost:3000#person
+
+<img src="example-01.png" />
+
```

Comment:
To be frank, I don't think we need this and it rather "pollutes" the root Readme. The Readme already covers the first time setup, the build and how to start the example applications. We don't need to explain how git and Github works within the JSON Forms Readme ;)

The only thing missing in the root Readme is on which port the example applications can be found. Note that it's port `8080` and not `3000` as here.

Reply:
You can either 'pollute' the readme or you can explain to stupid people like me how to build the project. As a JakartaEE developer I do not have the NodeJS expert knowledge as you expect...
I will remove the changes from the README again and continue working on the CSS fixes....

Reply:
Hi @rsoika, I'm pretty sure that you're not stupid at all, just unfamiliar with the tools used here. The root readme is just the wrong place for this kind of information / writing style. It would rather fit into a contribution guide or in the [wiki](https://github.com/eclipsesource/jsonforms/wiki/Developer-documentation). If you agree I would copy this over into the developer wiki.

Reply:
@sdirix , yes the wiki is maybe the best place for this kind of tutorial. In the mean time I added the information into my own readme [here](https://github.com/imixs/open-bpmn/tree/master/open-bpmn.glsp-client#jsonforms)